sql/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/view.sql

```diff
@@ -2,8 +2,39 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.telemetry.cohort_daily_statistics`
 AS
 SELECT
-  *
+  activity_date,
+  cohort_date,
+  normalized_app_name,
+  normalized_channel,
+  normalized_os,
+  normalized_os_version,
+  attribution_source,
+  attribution_medium,
+  attribution_campaign,
+  attribution_content,
+  attribution_experiment,
+  attribution_variation,
+  num_clients_active_on_day,
+  num_clients_in_cohort
```

Comment:
There are quite a few more dimensional columns.  Not that you'd need to include them all, but I'd think `country` would be good to have, and maybe also `locale`.

Also, since we're not including all the columns that make each row unique, I'd expect this to result in multiple rows with the same dimensional values.  Depending on how this view will be used, you might consider re-aggregating here on the dimensions and summing the measures (and same for the other query).

Reply:
Essentially, I'm trying to match what is already in Looker. Unfortunately, country isn't in the original view. Measure calculation also happens in Looker.